plugins/wiki/lib/foodsoft_wiki/wiki_parser.rb

```diff
@@ -9,10 +9,13 @@ class WikiParser < WikiCloth::Parser
       url_for page
     end
 
-    link_attributes_for do |page|
+    link_attributes_for do |page_anchor|
+      page, anchor = page_anchor.split(/#/, 2)
       permalink = Page.permalink(page)
-      if Page.exists?(:permalink => permalink)
-        { href: url_for(:wiki_page_path, permalink: permalink) }
+      if page.empty?
+        { href: '#' + anchor }
```

Comment:
what is `anchor` is `nil`?

Reply:
I did not find markup that produces that state.

I should note that markup such as 

```
[[|Custom label]]
```

still produces the following error message:

> translation missing: de.wikicloth_exception

This pull request is made to fix at least the typical use cases of anchors.

Reply:
With the latest commit here (b1fabe8), `anchor` is obviously non-`nil`, right?